package.json

```diff
@@ -23,7 +23,7 @@
     "build": "pnpm run -r clean && concurrently \"pnpm build:vite\" \"pnpm build:types\"",
     "dev": "concurrently \"pnpm build:vite --watch\" \"ts-node-esm .vite/server.ts\"",
     "release": "pnpm build",
-    "lint": "eslint --cache --ignore-path .gitignore . && pnpm lint:jison && prettier --cache --check .",
+    "lint": "eslint --cache --cache-strategy content --ignore-path .gitignore . && pnpm lint:jison && prettier --cache --check .",
     "lint:fix": "eslint --fix --ignore-path .gitignore . && prettier --write . && ts-node-esm scripts/fixCSpell.ts",
```

Comment:
Don't we need cache here too?

Reply:
Good idea! I didn't add it previously since I was worried about cache invalidation, but `--cache-strategy content` should mean that we have no issues.

PR in https://github.com/mermaid-js/mermaid/pull/4073
(hope you don't mind, I made you a co-author of the commit, since you suggested the idea :smile:)